subscribers:

  camera_topics: [
    "/webcam/image_raw",
    "/webcam2/image_raw"

  ]

  camera_frame_ids: [
    "camera",
    "camera2"
  ]

  queue_size: 1

publisher:

    topic: "/camera_signalman/image_raw"

    queue_size: 1
